Core xdr-retrieve-file different separator options (#45106)
demisto/content: Packs/CortexXDR/ReleaseNotes/6_3_33.md

## Cortex XDR by Palo Alto Networks

#### Integrations

##### Palo Alto Networks Cortex XDR - Investigation and Response

> - Added the *paths_separator* argument to the ***xdr-file-retrieve*** command, allowing retrieval of files whose paths contain a comma by specifying a different delimiter.

Reviewer: ```suggestion
- Added the ***paths_separator*** argument to the ***xdr-file-retrieve*** command to support custom delimiters for paths containing commas.
```
